index.html

```diff
@@ -17,7 +17,7 @@
 </head>
 
 <body>
-    <nav class="header">
```

Comment:
I had to remove the class name because we really don't need it unless we use CSS framework.